app/views/pre_registrations/edit.html.erb

```diff
@@ -1,132 +1,156 @@
-            </p>
-            <p>
-                <%= f.text_field :contactNumber ,:class => 'form-control', required: 'true',:maxlength=>"128"%>
-            </p>
-        </div>
+            <div class="col-md-6">
+                <%= f.label :user_id, 'Name of TTF' do %>
+                    <%= 'Name of TTF' %>
+                    <span class='required-asterisk'>*</span>
+                <% end %>
+                <p>
+                    <%= f.select :user_id, options_for_select(@ttf_list.collect { | ttf_list | [ttf_list.name,  ttf_list.id] }, selected: @pre_registration.user_id), {}, {required: 'true', autofocus: 'true', class: 'form-control', id:'select-other-projects'}%>
+                </p>
+            </div>
+                    
+            <div class="col-md-6">
+                <%= f.label :emailGroup, 'Suggested Email Group' do %>
+                    <%= 'Suggested Email Group' %>
+                    <span class='required-asterisk'>*</span>
+                <% end %>
+                <p>
+                    <%= f.text_field :emailGroup, :class => 'form-control', required: 'true', :maxlength=>"128", readonly: @flag ? true : false %>
+                </p>
+            </div>
 
-        <div class="col-md-6">
-            <p>
-                <%= label :personalEmail, 'Personal Email' %>
-                <%= label '', ' *', class: 'required-asterisk' %>
-            </p>
-            <p>
-                <%= f.text_field :personalEmail ,:class => 'form-control', required: 'true',:maxlength=>"128"%>
-            </p>
-        </div>
+            <div class="col-md-6">
+                <%= f.label :contactNumber, 'Contact Number' do %>
+                    <%= 'Contact Number' %>
+                    <span class='required-asterisk'>*</span> 
+                <% end %>
+                <p>
+                    <%= f.text_field :contactNumber ,:class => 'form-control', required: 'true', :maxlength=>"128", readonly: @flag ? true : false %>
+                </p>
+            </div>
 
-        <div class="col-md-6">
-            <p>
-                <%= label :holiday_scheme_id, 'Holiday Scheme' %>
-                <%= label '', ' *', class: 'required-asterisk' %>
-            </p>
-            <p>
-                <%= f.select :holiday_scheme_id, options_for_select(@holiday_scheme.collect { | holiday | [holiday.name,  holiday.id] }, selected: @pre_registration.holiday_scheme_id), {}, {required: 'true', autofocus: 'true', class: 'form-control', id:'select-other-projects'}%>
+            <div class="col-md-6">
+                <%= f.label :personalEmail, 'Personal Email' do %>
+                    <%= 'Personal Email' %>
+                    <span class='required-asterisk'>*</span>
+                <% end %>
+                <p>
+                    <%= f.text_field :personalEmail , :class => 'form-control', required: 'true',:maxlength=>"128", readonly: @flag ? true : false %>
```

Comment:
Space is missing after _required: 'true',_

Reply:
resolved